lib/livebook/application.ex

```diff
@@ -267,4 +268,9 @@ defmodule Livebook.Application do
       "Failed to start Livebook iframe server because port #{port} is already in use"
     )
   end
+
+  defp identity_provider() do
+    {module, key} = Livebook.Config.identity_provider()
+    [{module, name: LivebookWeb.ZTA, identity: [key: key]}]
```

Comment:
```suggestion
    [{module, name: LivebookWeb.ZTA, key: key}]
```

Reply:
Also, it doesn't seem we don't use the name?

Reply:
@jonatanklosko we will use in the future I think once we started fetching multiple fields (future PR)